coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -217,7 +217,10 @@ def get_bindings(alist) -> List[Any]:
 
         for i in alist:
             if isinstance(i, str):
-                results.append(context[i])
+                try:
+                    results.append(context[i])
+                except ValueError:
+                    results.append(None)
```

Comment:
This was the magic trick to continue when `bias` is None, because is an optional parameter.